### DISCUSSION TITLE
Add Indian languages in Translation.

### DISCUSSION THREAD

**Discussion by dhruvilxcode** [frappe/books#420] [General]
Hi, community, 
it will be wonderful if we add Indian Languages to translation files, we can start from one language for now.

it would be great if you can assign me the task. I am ready to contribute 🚀

**Comment by 18alantom** [comment]
Hey, thanks for offering to contribute.

You can check this wiki: https://github.com/frappe/books/wiki/Contributing-Translations for more info on how to contribute translations.

Let me know if you have any questions, I'd be more than happy to answer.

**Comment by dhruvilxcode** [comment]
Thanks, @18alantom, I read the contribution guide, fully ready to contribute, I will update the thread here, for all status updates.

**Comment by 18alantom** [comment]
@dhruvilxcode I'll be converting this into a discussion, you can update there.

**Reply by dhruvilxcode** [reply]
Thanks for moving into the discussion. I started working on it and generated translation files as per documentation. 🚀

**Reply by dhruvilxcode** [reply]
I am working on two Languages GU (Gujarati), HI (Hindi).

**Comment by dhruvilxcode** [comment]
Created PR for lang_code: gu 
[https://github.com/frappe/books/pull/423](https://github.com/frappe/books/pull/423)
